recipes/quest/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "quest" %}
+{% set version = "3.0-a1" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/erdc/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
@ceball This yaml file was taken from the example directory, if it needs to be host then it should also be changed there. Thanks

Reply:
I'm neither a conda nor a conda-forge person, I just happened to notice your PR because I have one in the queue too and I know quest from the earthsim project, so I wanted to help out :) So, don't take what I say as definitely correct...but I think maybe you have an out of date fork of staged-recipes; see https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L35. (Also, host is used in the docs, e.g. https://conda-forge.org/docs/meta.html#use-pip.)

Anyway, it probably makes no practical difference for a package like quest. (See e.g. https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#host for more info.)

Reply:
@ceball thank you for providing those resources and my fork may just be out of date, but since all the test have already passed with the current configuration then I think I will just leave it the way it is. I don't mind any free help what so ever ceball~!

Reply:
That should be host indeed, for pure python packages that will not be a problem. However, just to follow the right procedure here, let's change that to ``host``  :)

Reply:
Just for the next times, please do not delete the folders inside of the ``recipes``, because it may delete work which was already merged if the bot did catch that in time

Reply:
@marcelotrevisani I will keep that in mind, thanks.